MantidQt/CustomInterfaces/src/Indirect/IndirectDiffractionReduction.cpp

```diff
@@ -594,13 +594,13 @@ bool IndirectDiffractionReduction::validateRebin() {
     m_uiForm.valRebinStart->setText("");
     m_uiForm.valRebinWidth->setText("");
     m_uiForm.valRebinEnd->setText("");
-  } else {
+  }
+  else {
 #define CHECK_VALID(text, validator)                                           \
+  rebinValid = !text.isEmpty();												   \
```

Comment:
Now there is an extra call to `test.isEmpty()`, I think it is better left how it was